src/main/java/uk/gov/hmcts/juror/api/moj/service/expense/JurorExpenseServiceImpl.java

```diff
@@ -1069,17 +1078,18 @@ static class PendingApprovalMapId {
     @Override
     @Transactional
     @PreAuthorize(SecurityUtil.IS_MANAGER)
```

Comment:
Is this needed PreAuthorize annotation needed? Looks like it duplicates the check implemented in the controller

Reply:
This PreAuth was added to ensure if you enter this method outside of the controller layer it still validates you.
So an extra layer of security incase we re-use this method downstream